cmd/capabilities.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2022 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/util"
+	"github.com/spf13/cobra"
+)
+
+type capabilitiesParams struct {
+	capabilitiesFlag *capabilitiesFlag
+	showVersions     bool
+	showCurrent      bool
+}
+
+func newCapabilitiesParams() capabilitiesParams {
+	return capabilitiesParams{
+		capabilitiesFlag: newcapabilitiesFlag(),
+	}
+}
+
+func init() {
+
+	capabilitiesParams := newCapabilitiesParams()
+
+	var capabilitiesCommand = &cobra.Command{
+		Use:   "capabilities",
+		Short: "Print the capabilities of OPA",
+		Long: `Show capabilities for OPA.
+
+The 'capabilities' command prints the OPA capabilities for a specific version.
```

Comment:
Should we add "as known by this version" or "prior to and including the version of OPA used"?

Just because it might confuse people if they used, say, OPA 0.38.0, and there's been a release of 0.39.0, that wouldn't magically turn up in the list here.

Reply:
good point! [done](https://github.com/IoannisMatzaris/opa/blame/Expose-capabilities-through-cli-2/cmd/capabilities.go#L37)